packages/components/psammead-media-player/src/Placeholder/index.jsx

```diff
@@ -12,15 +14,48 @@ const StyledPlaceholder = styled.div`
   height: 100%;
 `;
 
-const Placeholder = ({ onClick, src }) => (
-  <StyledPlaceholder onClick={onClick}>
-    <Image alt="Image Alt" src={src} />
-  </StyledPlaceholder>
-);
+const StyledPlayButton = styled(PlayButton)`
+  position: absolute;
+  bottom: 0;
+
+  ${StyledPlaceholder}:hover &,
+  ${StyledPlaceholder}:focus & {
+    background-color: ${C_POSTBOX};
+  }
+`;
+
+const Placeholder = ({ onClick, service, src, mediaInfo }) => {
+  const { title, datetime, duration, durationSpoken, type } = mediaInfo;
+  return (
+    <StyledPlaceholder onClick={onClick}>
+      <StyledPlayButton
+        title={title}
+        service={service}
+        onClick={event => {
+          event.stopPropagation();
+          onClick();
+        }}
+        datetime={datetime}
+        duration={duration}
+        durationSpoken={durationSpoken}
+        type={type}
+      />
+      <Image alt="Image Alt" src={src} />
+    </StyledPlaceholder>
+  );
+};
 
 Placeholder.propTypes = {
   onClick: func.isRequired,
+  service: string.isRequired,
   src: string.isRequired,
+  mediaInfo: shape({
+    title: string.isRequired,
+    datetime: string,
+    duration: string,
+    durationSpoken: string,
+    type: oneOf(['video', 'audio']),
```

Comment:
Does this need a `.isRequired`? I know that in your play button component it defaults to video. But what happens if someone adds audio content and for some reason, the `type` isn't set.

Reply:
This is a good point - someone will need to look into this while I'm away. My concerns are:
- Requiring `type` in a component that doesn't use it directly
- Introducing inconsistency (requiring it in one and not the other)
- The relationship `type` has with `skin`. Could we infer the value of `type` from this and drop it altogether?
cc @jamesdonoh

Reply:
If it's not required in [psammead-play-button](https://github.com/bbc/psammead/tree/latest/packages/components/psammead-play-button) and defaults to `video`, I can't see why we'd need to make it required here. If the calling code fails to pass a `type` then AFAICS showing the standard play button (which visually at least could equally well apply to video or audio) is like a reasonable default.

And I'd be wary of coupling it to the `skin`, we shouldn't assume the type of the content and the skin being used will always go hand-in-hand.